### ISSUE TITLE
disabling SC2129 check should disable all connected lines

### ISSUE THREAD

**Issue by mika** [koalaman/shellcheck#440]
Given a file `foo` containing:

````
reprepro removematched bar '*foo' 2>>errors.txt                   
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
````

shellcheck complains regarding SC2129:

````
In foo line 1:
reprepro removematched bar '*foo' 2>>errors.txt
^-- SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects.
````

When disabling the SC2129 check via:

````
# shellcheck disable=SC2129
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
reprepro removematched bar '*foo' 2>>errors.txt
````

it complains about the next line(s) though:

````
In foo line 3:
reprepro removematched bar '*foo' 2>>errors.txt
^-- SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects.
````

It would be nice if `# shellcheck disable=SC2129` could disable those checks for all connected lines, so you don't have to add it before every single line.

Thanks for shellcheck! :+1:

**Comment by koalaman** [comment]
That is indeed annoying. As a workaround, you could ignore them as a block:

    # shellcheck disable=SC2129
    {
      reprepro removematched bar '*foo' 2>>errors.txt                   
      reprepro removematched bar '*foo' 2>>errors.txt
      reprepro removematched bar '*foo' 2>>errors.txt
      reprepro removematched bar '*foo' 2>>errors.txt
      reprepro removematched bar '*foo' 2>>errors.txt
    }

**Comment by koalaman** [comment]
This has been fixed in 26d16eb

**Comment by mika** [comment]
Wow, thanks a lot for the fast fix! Highly appreciated! :+1:
